tx-source/site_core.php

```diff
@@ -1930,3 +1930,8 @@
 // Quotes/Formats
 $Definition['Toggle Quote'] = 'Toggle Quote';
 $Definition['Spoiler Warning'] = 'Spoiler Warning';
+
+// Catalogue Display Plugin/Formats
+$Definition['Each discussion will show an uploaded image on the Category page. This only applies to categories with "Discussions" as the "Display As."'] = 'Each discussion will show an uploaded image on the Category page. This only applies to categories with "Discussions" as the "Display As."';
```

Comment:
Don't use long strings as keys. See [our guidance on long strings](https://staff.vanillaforums.com/kb/articles/225-guide-for-adding-and-editing-translation-strings#long-strings).

Reply:
This also seems like a dashboard string. In that case you need to put it in dash_core or dash_text.